build-tools/xaprepare/xaprepare/ConfigAndData/Runtimes.cs

```diff
@@ -389,6 +389,8 @@ partial class Runtimes
 			new MonoUtilityFile ("monodoc.dll.config",                     ignoreDebugInfo: true),
 			new MonoUtilityFile ("mono-api-html.exe",                      remap: true),
 			new MonoUtilityFile ("mono-api-info.exe",                      remap: true),
+			new MonoUtilityFile ("pdb2mdb.exe",                       	   remap: true),
```

Comment:
Looks like there are mixed spaces/tabs:
```suggestion
			new MonoUtilityFile ("pdb2mdb.exe",                            remap: true),
```